src/components/Icon/index.js

```diff
@@ -22,6 +28,30 @@ const Loading = ({
   )
 }
 
+const typeToImageMapping = {
+  'empty-search': emptySearch,
+  'empty-new-conversation': emptyNewConversation,
+  'api-error': apiError,
+  'empty-folder': emptyFolder,
+  'active-conversation': activeConversation,
+  'add-notes': addNotes,
+}
+
+const Custom = ({ type, message }) => {
+  if (typeToImageMapping[type]) {
+    return (
+      <div className="sg contacto-image-wrapper">
+        <div className="contacto-image">
+          <img src={typeToImageMapping[type]} alt="style-guide-text.png" />
```

Comment:
Can we update the alt text to the message? or some other meaningful text.

Reply:
sure